test/suites/dev/moonbase/test-randomness/test-randomness-vrf-lottery5.ts

```diff
@@ -48,7 +48,7 @@ describeSuite({
         args: [0],
       });
 
-      expect(estimatedGas).toMatchInlineSnapshot(`284196n`);
```

Comment:
Cannot explain why this test changes. The new precompile in theory should not affect it.

Reply:
Probably related to the `Cargo.lock`